tests/Unit/Response/DetectAndFeaturesResponseTest.php

```diff
@@ -114,13 +111,11 @@ public function malformedResponseWithNotCompleteDetections(): void
         ];
 
         $this->expectException(MalformedResponse::class);
-        $this->expectErrorMessageMatches('#^Undefined#');
+        $this->expectExceptionMessage('Expected a value other than null.');
```

Comment:
why it's different?

Reply:
Because of updating phpunit.
In previous version Undefined Index was thrown as an error.
In new version undefined index is an notice, and default null is return.

\Answear\WideEyesBundle\ValueObject\BoundingBox::22 `y2` is now default null (even if it's undefined) and assertNull is throwing `Expected a value other than null.`